packages/sources/finage-test/src/endpoint/crypto-router.ts

```diff
@@ -1,13 +1,13 @@
 import { CryptoPriceEndpoint } from '@chainlink/external-adapter-framework/adapter'
```

Comment:
Think we just need to remove `-router` from `finage-test` endpoints then this PR is good to go

Reply:
done